pkg/plugins/network/modules.go

```diff
@@ -50,6 +50,42 @@ func (d *DeployNetworkPluginModule) Init() {
 	default:
 		return
 	}
+	if d.KubeConf.Cluster.Network.EnableMultusCNI {
+		d.Tasks = append(d.Tasks, deployMultus(d)...)
+	}
+}
+
+func deployMultus(d *DeployNetworkPluginModule) []task.Interface {
+	generateMultus := &task.RemoteTask{
+		Name:  "GenerateMultus",
+		Desc:  "Generate multus cni",
+		Hosts: d.Runtime.GetHostsByRole(common.Master),
+		Prepare: &prepare.PrepareCollection{
+			new(common.OnlyFirstMaster),
+			&OldK8sVersion{Not: true},
```

Comment:
If the `multus-cni` support below k8s v1.16 version, maybe this `OldK8sVersion prepare` can be deleted.

Reply:
https://github.com/kubesphere/kubekey/blob/master/README.md#kubernetes-versions

kubekey not support below k8s v1.17.9 version,

Reply:
In fact, kk support installing k8s v1.15.12. But naturally, we do not suggest installing this k8s version.

https://github.com/kubesphere/kubekey/blob/master/docs/kubernetes-versions.md

Reply:
`multus-cni:v3.8`  CRD use `apiextensions.k8s.io/v1`  ,k8s v1.15.12  not support.

Reply:
Thanks for your research. It explains my confusion.